src/Circlereader/util/optionsAttribute.ts

```diff
@@ -0,0 +1,631 @@
+      value: ActivationFunctionType[addOpt.fusedActivationFunction()]
+    });
+  }
+
+  static getL2NormAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let l2NormOpt = new L2NormOptions();
+    l2NormOpt = operator.builtinOptions<flatbuffers.Table>(l2NormOpt);
+
+    attributes.push({
+      attribute: 'fused_activation_function',
+      value: ActivationFunctionType[l2NormOpt.fusedActivationFunction()]
+    });
+  }
+
+  static getLocalResponseNormalizationAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let localResponseNomalizationOpt = new LocalResponseNormalizationOptions();
+    localResponseNomalizationOpt =
+        operator.builtinOptions<flatbuffers.Table>(localResponseNomalizationOpt);
+
+    attributes.push({attribute: 'radius', value: localResponseNomalizationOpt.radius()});
+    attributes.push({attribute: 'bias', value: localResponseNomalizationOpt.bias()});
+    attributes.push({attribute: 'alpha', value: localResponseNomalizationOpt.alpha()});
+    attributes.push({attribute: 'beta', value: localResponseNomalizationOpt.beta()});
+  }
+
+  static getLSTMAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let lstmOpt = new LSTMOptions();
+    lstmOpt = operator.builtinOptions<flatbuffers.Table>(lstmOpt);
+
+    attributes.push({
+      attribute: 'fused_activation_function',
+      value: ActivationFunctionType[lstmOpt.fusedActivationFunction()]
+    });
+    attributes.push({attribute: 'cell_clip', value: lstmOpt.cellClip()});
+    attributes.push({attribute: 'proj_clip', value: lstmOpt.projClip()});
+    attributes.push({attribute: 'kernel_type', value: LSTMKernelType[lstmOpt.projClip()]});
+    attributes.push(
+        {attribute: 'asymmetric_quantize_inputs', value: lstmOpt.asymmetricQuantizeInputs()});
+  }
+
+  static getResizeBilinearAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let resizeBilinearOpt = new ResizeBilinearOptions();
+    resizeBilinearOpt = operator.builtinOptions<flatbuffers.Table>(resizeBilinearOpt);
+
+    attributes.push({attribute: 'align_corners', value: resizeBilinearOpt.alignCorners()});
+    attributes.push({attribute: 'half_pixel_centers', value: resizeBilinearOpt.halfPixelCenters()});
+  }
+
+  static getCallAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let callOpt = new CallOptions();
+    callOpt = operator.builtinOptions<flatbuffers.Table>(callOpt);
+
+    attributes.push({attribute: 'subgraph', value: callOpt.subgraph()});
+  }
+
+  static getReshapeAttr(operator: Operator, attributes: Array<NodeAttributes>) {
+    let reshapeOpt = new ReshapeOptions();
+    reshapeOpt = operator.builtinOptions<flatbuffers.Table>(reshapeOpt);
+
+    let newShapeLength = reshapeOpt.newShapeLength();
+    attributes.push({attribute: 'new_shape_length', value: newShapeLength});
+
+    for (let i = 0; i < newShapeLength; i++) {
+      attributes.push({attribute: 'new_shape_' + i, value: reshapeOpt.newShape(i)});
```

Comment:
Maybe `List` type should be pushed...?

Reply:
### As-is

```
{new_shape_0, value0},
{new_shape_1, value1},
...
```

### To-be
```
{newShape, [value0, value1, ...]}
```

Reply:
`{newShape, [value0, value1, ...]}`

Is this what you mean?

Reply:
Ah, yes it is :)